lib/compress.js

```diff
@@ -1799,16 +1804,6 @@ merge(Compressor.prototype, {
         }
     };
 
-    OPT(AST_While, function(self, compressor) {
-        if (!compressor.option("loops")) return self;
-        self = AST_DWLoop.prototype.optimize.call(self, compressor);
-        if (self instanceof AST_While) {
-            if_break_in_loop(self, compressor);
```

Comment:
What is the equivalent logic for `if_break_in_loop(self, compressor);` in the new `OPT(AST_DWLoop)` code above? I see a newly added `else` clause for the `self instanceof AST_Do` case, but the `AST_While` logic is the same as it was. Was `if_break_in_loop` never needed?